portafly/src/i18n/locales/en/account_settings/personal/personal_details.yml

```diff
@@ -1,5 +1,74 @@
+
+# Main area
+
+# Tabs
+
+user_data_title: User data
+user_data_description: This is the title of the form containing the user's personal information.
+change_password_title: Change password
+change_password_description: This is the title of the form containing the user's current password field and new password field.
+
+# User data form
+
+username_label: Username
+username_label_description: Label for the field that holds the current username.
+username_value: {{current_username}}
+username_value_description: This is the value contained in the username field.
+username_validation: {{username_validation}} is too short. Use a minimum of 3 characters. Do not use spaces. You can include letters, numbers, and the following special characters . - _ @ +
+username_validation_description: Input field that holds the current username, and validates the content.
+
+email_label: Email
+email_label_description: Label for the field that holds the valid user's email address.
+email_value: {{current_user_email}}
+email_value_description: By default, this is the email of the current user.
+email_validation: Use a valid email such as 'admin@domain.com'. The email can contain letters (both lower and upper case), numbers, and the following characters . - _ @ +
+email_validation_description: Input field that holds the valid user's email address.
+
+first_name_label: First name
+first_name_label_description: Label for the field that holds the user's first name.
+first_name_value: [User first name]
+first_name_value_description: Input field that holds the user's first name.
+
+last_name_label: Last name
+last_name_label_description: Label for the field that holds the user's last name.
+last_name_value: [User last name]
+last_name_value_description: Input field that holds the user's last name.
+update_details_button: Update details
+update_details_description: The user clicks this button to procede with updating their details.
+
+# Modal: Update details
+
+current_password_title: Provide your current password to update your personal details.
+current_password_title_description: Body title of modaldialog prompts the user to provide their current password to make the update.
+current_password_label: Current password
+current_password_label_description: Label for the field for the user to enter a new password.
+current_password_value: [User password]
+current_password_value_description: Input field that holds the user's password.
+current_password_helper: When you update your personal details, all other active sessions for this user will end. This happens in order to confirm that it is you, and not somebody else trying to lock you out of your account.
+current_password_helper_description: Text to let the user know what happens when they update their personal details.
+confirm_button: Confirm
+confirm_button_discription: The user clicks this button to confirm that the current password is entered in the field currectly.
```

Comment:
```suggestion
confirm_button_description: The user clicks this button to confirm that the current password is entered in the field currectly.
```

Reply:
@dfennessy Perhaps you should consider adding this string to `shared.yml`, with a generic description. Example:
`confirm_button_description: The user clicks this button to confirm the action described in the modal.`